Building clang container failed with this error:
```
==> Error: ProcessError: Command exited with status 2:
    'make' '-j2'

2 errors found in build log:
     1     ==> pigz: Executing phase: 'edit'
     2     ==> [2022-09-07-19:16:46.811453] Using default implementation: skipp
           ing edit phase.
     3     ==> pigz: Executing phase: 'build'
     4     ==> [2022-09-07-19:16:46.833180] 'make' '-j2'
     5     gcc -O3 -Wall -Wextra -Wno-unknown-pragmas -Wcast-qual   -c -o pigz.
           o pigz.c
     6     gcc -O3 -Wall -Wextra -Wno-unknown-pragmas -Wcast-qual   -c -o yarn.
           o yarn.c
  >> 7     pigz.c:415:10: fatal error: zlib.h: No such file or directory
     8       415 | #include "zlib.h"       // deflateInit2(), deflateReset(), d
           eflate(),
     9           |          ^~~~~~~~
     10    compilation terminated.
  >> 11    make: *** [<builtin>: pigz.o] Error 1
     12    make: *** Waiting for unfinished jobs....

See build log for details:
  /tmp/root/spack-stage/spack-stage-pigz-2.7-fhhp3sk4w6ipgtmjsc7cdinvbpuaqe6x/spack-build-out.txt
```

--- a/.gitmodules
+++ b/.gitmodules
@@ -1,9 +1,11 @@
 [submodule "spack"]
   path = spack
-  #url = https://github.com/spack/spack
-  #branch = develop
-  url = https://github.com/NOAA-EMC/spack
-  branch = jcsda_emc_spack_stack
+  ##url = https://github.com/spack/spack
+  ##branch = develop
+  #url = https://github.com/NOAA-EMC/spack
+  #branch = jcsda_emc_spack_stack
+  url = https://github.com/climbfuji/spack
+  branch = feature/add_xarray_f90nml
 [submodule "doc/CMakeModules"]
   path = doc/CMakeModules
   url = https://github.com/noaa-emc/cmakemodules

--- a/configs/containers/docker-ubuntu-clang-mpich.yaml
+++ b/configs/containers/docker-ubuntu-clang-mpich.yaml
@@ -166,8 +166,9 @@ spack:
         ln -svf libc++abi.so.1.0 libc++abi.so
       pre_final: |
         #Set environment variables for installing tzdata
-        ENV DEBIAN_FRONTEND=noninteractive && \
-        TZ=Etc/UTC && \
+        ENV DEBIAN_FRONTEND=noninteractive
+        ENV TZ=Etc/UTC
+      final: |
         #Create symbolic links for clang compilers
         RUN cd /usr/bin && \
         ln -svf clang-10 clang && \
@@ -175,7 +176,6 @@ spack:
         ln -svf clang-cpp-10 clang-cpp && \
         cd /usr/lib/llvm-10/lib && \
         ln -svf libc++abi.so.1.0 libc++abi.so
-      final: |
         #Make a non-root user:nonroot / group:nonroot for running MPI
         RUN useradd -U -k /etc/skel -s /bin/bash -d /home/nonroot -m nonroot --uid 43891 && \
         echo "ulimit -s unlimited" >> /home/nonroot/.bashrc && \

--- a/configs/containers/docker-ubuntu-gcc-openmpi.yaml
+++ b/configs/containers/docker-ubuntu-gcc-openmpi.yaml
@@ -145,8 +145,8 @@ spack:
     extra_instructions:
       pre_final: |
         #Set environment variables for installing tzdata
-        ENV DEBIAN_FRONTEND=noninteractive &&\
-        TZ=Etc/UTC
+        ENV DEBIAN_FRONTEND=noninteractive
+        ENV TZ=Etc/UTC
       final: |
         #Make a non-root user:nonroot / group:nonroot for running MPI
         RUN useradd -U -k /etc/skel -s /bin/bash -d /home/nonroot -m nonroot --uid 43891 && \

--- a/configs/templates/skylab-dev/spack.yaml
+++ b/configs/templates/skylab-dev/spack.yaml
@@ -70,10 +70,13 @@ spack:
     - odc@1.4.5
     - parallel-netcdf@1.12.2
     - parallelio@2.5.4
+    - py-f90nml@1.4.2
     - py-numpy@1.22.3
     - py-pandas@1.4.0
+    - py-pyyaml@6.0
     - py-scipy@1.8.0
     - py-shapely@1.8.0
+    - py-xarray@2022.3.0
     # DH* fake version number
     - r2d2@0.0.1
     # DH* fake version number

